src/api.ts

```diff
@@ -5,6 +5,8 @@ import { User } from './types/User';
 const BASE_URL =
   'https://mate-academy.github.io/react_dynamic-list-of-todos/api';
```

Comment:
The `BASE_URL` is split across two lines. Consider combining it into a single line for better readability, unless there is a specific reason for this formatting.